be/src/vec/exec/ves_http_scan_node.cpp

```diff
@@ -124,6 +124,12 @@ Status VEsHttpScanNode::open(RuntimeState* state) {
     SCOPED_CONSUME_MEM_TRACKER(mem_tracker());
     RETURN_IF_CANCELLED(state);
 
+    std::map<std::string, std::string> properties(_properties);
+    if (properties.find(ESScanReader::KEY_QUERY_DSL) != properties.end()) {
```

Comment:
Why not using `_properties` directly?

Reply:
a little confused. Maybe you should add a comment here to point out `KEY_QUERY_DSL` in properties means we turn `enable_new_es_dsl` on in FE and we have already removed push-down predicates here.

Reply:
> a little confused. Maybe you should add a comment here to point out `KEY_QUERY_DSL` in properties means we turn `enable_new_es_dsl` on in FE and we have already removed push-down predicates here.

In order to pass only one field, the argument logic is controlled in FE, so let me comment it.